src/common/deflate.h

```diff
@@ -57,6 +57,20 @@ static const int kWindowBitsMaxRaw = -kWindowBitsMax;
 byte *decompressDeflate(const byte *data, size_t inputSize,
                         size_t outputSize, int windowBits);
 
+/** Decompress (inflate) using zlib's DEFLATE algorithm without knowing the output size.
+ *
+ *  @param  data       The compressed input data.
+ *  @param  inputSize  The size of the input data in bytes.
+ *  @param  outputSize The size of the decompressed output data.
+ *  @param windowBits  The base two logarithm of the window size (the size of
+ *                     the history buffer). See the zlib documentation on
+ *                     inflateInit2() for details.
+ *  @param frameSize   The size of the extracted frames, defaults to 4096.
+ *  @return The decompressed data
+ */
+byte *decompressDeflateWithoutOutputSize(const byte *data, size_t inputSize, size_t &outputSize,
+						 int windowBits, unsigned int frameSize = 4096);
```

Comment:
The alignment is off here now

Reply:
The name is more descriptive now, yeah. It's a bit clunky, but it's better than the first one :)